package.json

```diff
@@ -44,7 +44,7 @@
     "lodash": "^3.10.0",
     "method-override": "^2.3.3",
     "mocha": "~1.20.0",
-    "mongoose": "^4.0.6",
+    "mongoose": "4.1.11",
```

Comment:
Definitely, but looking into their commit history maybe this https://github.com/Automattic/mongoose/commit/9af0d5fb0c5ffd4f8382715a1837e3c110b4ed3b